corehq/apps/app_manager/tests/data/days_ago_suite.xml

```diff
@@ -1,220 +1,116 @@
 
-
 <suite version="1" descriptor="Suite File">
-
-
-
 <xform>
     <resource id="9473e9dee517a6387fbb5698a3cfdd3a669d17c9" version="1">
         <location authority="local">./modules-0/forms-0.xml</location>
         <location authority="remote">./modules-0/forms-0.xml</location>
     </resource>
 </xform>
-
-
-
-
 <locale language="default">
     <resource id="app_default_strings" version="1">
         <location authority="local">./default/app_strings.txt</location>
         <location authority="remote">./default/app_strings.txt</location>
     </resource>
 </locale>
-
 <locale language="en">
     <resource id="app_en_strings" version="1">
         <location authority="local">./en/app_strings.txt</location>
         <location authority="remote">./en/app_strings.txt</location>
     </resource>
 </locale>
-
-
-
-
-
-
 <detail id="m0_case_short">
     <title><text><locale id="m0.case_short.title"/></text></title>
-
-
     <field sort="default">
         <header>
             <text><locale id="m0.case_short.case_dob_1.header"/></text>
         </header>
-
+        <template width="0">
+        <text>
+          <xpath function="dob"/>
+        </text>
+      </template>
+    </field>
+    <field>
+        <header width="0">
+            <text/>
+        </header>
```

Comment:
This is because these tests use commcare pre-2.2, at which point we still had to mock sorting by something other than the display, by having a sort column (with zero-width cells) and a display column (with zero-width headers).